Jenkinsfile

```diff
@@ -7,18 +7,8 @@
 import com.sonatype.jenkins.pipeline.GitHub
 import com.sonatype.jenkins.pipeline.OsTools
 
-properties([
-  parameters([
-    string(defaultValue: '', description: 'New Nexus Repository Manager Version', name: 'nexus_repository_manager_version'),
-    string(defaultValue: '', description: 'New Nexus Repository Manager Version Sha256', name: 'nexus_repository_manager_version_sha'),
-    string(defaultValue: '', description: 'New Nexus Repository Manager Cookbook Version', name: 'nexus_repository_manager_cookbook_version'),
-    booleanParam(defaultValue: false, description: 'Skip Pushing of Docker Image and Tags', name: 'skip_push'),
-    booleanParam(defaultValue: false, description: 'Only update the latest tag', name: 'update_latest_only')
-  ])
-])
```

Comment:
Everything removed from this file is preserved in the new Jenkinsfile-Release file. This one will only be for slimmer snapshot builds of feature and master branches.

Reply:
Does the `Jenkinsfile-Release` file contain everything that used to be in the original `Jenkinsfile`? It's hard to tell.

Reply:
I'm ambivalent about it aside from suggesting it as a way for `git` to understand what happened. :)

I believe `git` only recognizes moves heuristically so unfortunately using `git mv` first won't help once we squash.

Reply:
I tried it locally, `git mv Jenkinsfile Jenkinsfile-Release` and then `cp Jenkinsfile-Release Jenkinsfile` back, but once the files were added, git lost the trail:
```
jflinchbaugh@Johns-MacBook-Pro:~/workspace/docker-nexus3
$ g s
On branch mv-test
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	renamed:    Jenkinsfile -> Jenkinsfile-Release

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	Jenkinsfile

jflinchbaugh@Johns-MacBook-Pro:~/workspace/docker-nexus3
$ g add .
jflinchbaugh@Johns-MacBook-Pro:~/workspace/docker-nexus3
$ g s
On branch mv-test
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   Jenkinsfile
	new file:   Jenkinsfile-Release
```